src/documents/tests/test_migration_document_pages_count.py

```diff
@@ -0,0 +1,58 @@
+import os
+import shutil
+
+from django.conf import settings
+
+from documents.tests.utils import TestMigrations
+
+
+def source_path_before(self):
+    if self.filename:
+        fname = str(self.filename)
+
+    return os.path.join(settings.ORIGINALS_DIR, fname)
+
+
+class TestMigrateDocumentPagesCount(TestMigrations):
+    migrate_from = "1052_document_transaction_id"
+    migrate_to = "1053_document_pages_count"
+
+    def setUpBeforeMigration(self, apps):
+        Document = apps.get_model("documents", "Document")
+        doc = Document.objects.create(
+            title="test1",
+            mime_type="application/pdf",
+            filename="file1.pdf",
+        )
+        self.doc_id = doc.id
+        shutil.copy(
+            os.path.join(os.path.dirname(__file__), "samples", "simple.pdf"),
```

Comment:
It's a nit, but let's keep using `Path` when creating new code

Reply:
> It's a nit, but let's keep using `Path` when creating new code

Indeed, I hadn't noticed that Path was sometimes used.

Reply:
Updated : [Refactor file copy to use Path instead of os.path.join](https://github.com/paperless-ngx/paperless-ngx/pull/7750/commits/a85d6cdc3831ddf2f076905fe8ea107f1509a3d1)